instrumentation/cassandra/cassandra-3.0/src/main/java/io/opentelemetry/auto/instrumentation/cassandra/v3_0/CassandraClientInstrumentation.java

```diff
@@ -35,7 +36,7 @@
 public class CassandraClientInstrumentation extends Instrumenter.Default {
 
   public CassandraClientInstrumentation() {
-    super("cassandra");
+    super(DbSystem.CASSANDRA);
```

Comment:
since not really tied to DbSystem constant
```suggestion
    super("cassandra");
```